app/submitter/converter.py

```diff
@@ -89,7 +89,7 @@ def convert_answers_to_data(answer_store, questionnaire_json, routing_path):
         answer_schema_list = SchemaHelper.get_answers_by_id_for_block(block_json)
 
         for answer in answers_in_block:
-            answer_schema = answer_schema_list[answer['answer_id']]
+            answer_schema = answer_schema_list.get(answer.get('answer_id', ''), None)
```

Comment:
A little unreadable. Perhaps change to the following to keep it simple?

```python
try:
    answer_schema = answer_schema_list[answer['answer_id']]
except KeyError:
    answer_schema = None
```

Reply:
i thought the get might be better but this does look more readable